docs/Configuration/Overview.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Configuration Overview
+permalink: /configuration/overview
+category: Configuration
+menuOrder: 1
+---
+
+Cube.js is designed as an extremely configurable and flexible microservice.
+This flexibility allows you to tune Cube.js to your exact requirements.
```

Comment:
As a general principle, I suggest that we refrain from using "estimative" adjectives (such as "extremely" here) in any written content because...

1) they don't really convey a certain meaning because everyone perceive their meaning subjectively (for some people it is extremely configurable, others may think it has suboptimal configurability)

2) if non-trivial facts are described (often the case), such claims need to be supported with evidence and proofs  (see [Show, don't tell](https://en.wikipedia.org/wiki/Show,_don%27t_tell))

Then why bother to use such adjectives of we can be specific and provide evidence in the first place? :)

An example:

```suggestion
Cube.js is designed to work with different configuration sources.
```